### PR TITLE
Test Files for AddPlanCommand and ListPlanCommand

### LINKED ISSUE #33: Implement "add-plan" command
- Implement "add plan" command
- Implement test cases

### PR DESCRIPTION
**Pull request by victorlaiyeeteng**
Closes #33

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -63,71 +63,69 @@ FriendBook is a **desktop app for managing contacts, optimized for use via a Com
...
-- Additional arguments provided: `Invalid syntax: Too many arguments. Syntax: add friend n/NAME p/PHONE_NUMBER e/EMAIL`
+- Additional arguments provided: `Invalid syntax: Too many arguments. Syntax: add-friend n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS`
 
-### Removing a Friend : `remove friend` [Coming Soon]
+### Removing a Friend : `delete-friend`
 
 Removes an existing friend from the friends list.
 
-Format: `remove friend NAME`
+Format: `delete-friend NAME`
```
Comment (jannnice) [inline docs/UserGuide.md]: format should be: delete-friend INDEX

Comment (victorlaiyeeteng) [reply docs/UserGuide.md]: Fixed! Thank you!

**Comment by Elijah5399** [comment]
LGTM

**Review by Elijah5399 — APPROVED** [review]

**Comment by jannnice** [comment]
ok goooood

**Review by jannnice — APPROVED** [review]
ok goooooood

**Pull request merged**